### ISSUE TITLE
Incorrect semantics used on the skip link

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2013]
## Bug Report

**Problematic behavior**
There is indeed a skip link but semantically it's a button.

**Expected behavior/code**
The skip link view content must be structured in an <a> element and not a <button type="button">

**Steps to Reproduce**
1. Log in to docs
2. Tab with the keyboard
3. And then the bug happens!

**Additional context/Screenshots**

<img width="383" height="128" alt="Image" src="https://github.com/user-attachments/assets/58403a9c-d6d5-4e6c-a5df-5ded28f1fef5" />

**Critères RGAA**

**Critère 7.1** : [[Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 

A **link** is used for navigation (navigating between different pages / websites / anchors on the same page or a file to download)

A **button** is used to trigger an action or event (submit a form / open a modal, show/hide content)

It won't be announced in the same way, and navigation doesn't work the same way either. A link is activated with the `enter` key, a button with the `enter` and `space` keys

And finally, screen readers read pages in a linear fashion but also by list. If some users navigate by "list" of links or buttons on the page. If the wrong element is used, they won't find what they're looking for in that list.

**Comment by FocusMode-coder** [comment]
Hey! I may be able to help with this bug.

I build custom fixes for Python projects and automation systems.

If you'd like help resolving this issue quickly feel free to reach out.

Happy to take a look 👍
